core/src/main/java/org/bitcoinj/core/Transaction.java

```diff
@@ -1811,7 +1835,6 @@ public void setMemo(String memo) {
      * traditional manner without witness data, and t is the size of a transaction serialized in the segwit format
      * with witness data.
      */
-    // TODO: write tests for this
```

Comment:
I mistook weight for cost. We do not have a test yet for weight. I reinstated the TODO.